PhoneNumberKit/UI/TextField.swift

```diff
@@ -180,6 +205,52 @@ open class PhoneNumberTextField: UITextField, UITextFieldDelegate {
         super.delegate = self
     }
 
+    open func exampleNumber(for countryCode: String) -> String {
```

Comment:
would prefer that this be named to as `placeholderNumber(for countryCode: String)` …

or … perhaps even better … the name can remain, but, since, in fact, the logic of this function does not really appear to be related to an instance of `TextField`, but rather to the business logic of what an example placeholder should be.  perhaps it would be best if this function were moved to the same class that provides `phoneNumberKit.metdata(for:)` and `phoneNumberKit.parse(_:withRegion:ignoreType)` …

Reply:
Completely agree, I've made this change and added 2 variants one that returns a `PhoneNumber` and one that returns a formatted `String` of the `PhoneNumber`